src/utils/getDefaultCommitContent.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import fs from 'fs'
+
+const COMMIT_FILE_PATH_INDEX = 3
+const COMMIT_TITLE_LINE_INDEX = 0
+const COMMIT_MESSAGE_LINE_INDEX = 2
+
+const getDefaultCommitContent = () => {
+  /*
+    Since the hook is called with `gitmoji --hook $1`
+    According to https://git-scm.com/docs/githooks#_prepare_commit_msg,
+    the commit file path will be stored in the 4th argument of the command
+  */
+  const commitFilePath: string = process.argv[COMMIT_FILE_PATH_INDEX]
+
+  if (fs.existsSync(commitFilePath) === false) {
+    return {
+      title: undefined,
+      message: undefined
+    }
+  }
```

Comment:
This is for when the hook command it's not autocompleted right ? Can we use for example `null` instead of `undefined` in here to remove the default values for inquirer prompt ?

I think we should use `null` instead of `undefined`

Reply:
Done